src/egpg.sh

```diff
@@ -125,6 +125,10 @@ call_gpg() {
     local file=$1; shift
     local pyfile="$LIBDIR/gpg/$file"
     [[ -f "$pyfile" ]] || fail "Cannot find python file: $pyfile"
+    if is_true $DEBUG; then
+        # User can override level by exporting GPGME_DEBUG
+        [[ -z "$GPGME_DEBUG" ]] && export GPGME_DEBUG=2
+    fi
```

Comment:
A shorter way to do this is also:
`is_true $DEBUG && export GPGME_DEBUG=${GPGME_DEBUG:-2}`
But this is also more cryptic. What you have done is more readable.

Reply:
Thanks, will look into it might be it might be helpful somewhere else.  I am merging this for now.